sbin/ipf/ipf/ipf.5

```diff
@@ -295,7 +295,7 @@ proto-unr
 (protocol unreachable)
 the IP protocol specified in the packet is not available to receive
 packets.
-.DE
+.RE
```

Comment:
There's no `.DE` in man(7).

Given the context, I think this should be `.RE` to close the `.RS` on line 257.